plugin/pkg/scheduler/algorithm/predicates/predicates.go

```diff
@@ -108,31 +109,44 @@ type resourceRequest struct {
 func getResourceRequest(pod *api.Pod) resourceRequest {
 	result := resourceRequest{}
 	for ix := range pod.Spec.Containers {
-		limits := pod.Spec.Containers[ix].Resources.Limits
-		result.memory += limits.Memory().Value()
-		result.milliCPU += limits.Cpu().MilliValue()
+		requests := pod.Spec.Containers[ix].Resources.Requests
+		result.memory += requests.Memory().Value()
+		result.milliCPU += requests.Cpu().MilliValue()
 	}
 	return result
 }
 
-func CheckPodsExceedingCapacity(pods []*api.Pod, capacity api.ResourceList) (fitting []*api.Pod, notFitting []*api.Pod) {
+func CheckPodsExceedingCapacity(pods []*api.Pod, capacity api.ResourceList, maxPods int64) (fitting []*api.Pod, notFitting []*api.Pod) {
```

Comment:
Yeah the whole eviction may be too contentious for now, let's see what @davidopp and @bgrant0607 think